website/source/guides/operations/performance-nodes.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: "guides"
+page_title: "Performance Standby Nodes - Guides"
+sidebar_current: "guides-operations-performance-nodes"
+description: |-
+  This guide will walk you through a simple Vault Highly Available (HA) cluster
+  implementation. While this is not an exhaustive or prescriptive guide that
+  can be used as a drop-in production example, it covers the basics enough to
+  inform your own production setup.
+---
+
+# Performance Standby Nodes
+
+~> **Enterprise Only:** Performance Standby Nodes feature is a part of _Vault Enterprise_.
+
+In [Vault High Availability](/guides/operations/vault-ha-consul.html) guide, it
+was explained that only one Vault server will be _active_ in a cluster and
+handles **all** requests (reads and writes).  The rest of the servers become the
+_standby_ nodes and simply forward requests to the _active_ node.  
+
+![HA Architecture](/assets/images/vault-ha-consul-3.png)
+
+If you are running **_Vault Enterprise_ 0.11** or later, those standby nodes can
+handle most read-only requests (encryption/decryption of data using [transit](/docs/secrets/transit/index.html)
+keys, GET requests of key/value secrets) and behave as read-replica. This can
```

Comment:
If you are running **_Vault Enterprise_ 0.11** or later, those standby nodes can
handle most read-only requests. For example performance standbys can handle encryption/decryption of data using [transit](/docs/secrets/transit/index.html) keys, GET requests of key/value secrets and other requests that do not change underlying storage.